app/src/main/java/com/mobilecoronatracker/ui/about/FragmentAbout.kt

```diff
@@ -0,0 +1,47 @@
+package com.mobilecoronatracker.ui.about
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.databinding.DataBindingUtil
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.Observer
+import com.mobilecoronatracker.R
+import com.mobilecoronatracker.databinding.FragmentAboutBinding
+import kotlinx.android.synthetic.main.fragment_about.*
+import com.mobilecoronatracker.ui.utils.addLinksNoAppend
+import java.util.regex.Pattern
+
+class FragmentAbout : Fragment() {
+    private lateinit var viewModel: CreditsListViewModel
+    private val adapter = CreditsListAdapter()
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        val binding = DataBindingUtil.inflate<FragmentAboutBinding>(inflater,
+            R.layout.fragment_about, container, false
+        )
+        viewModel = CreditsListViewModel(activity!!.application)
```

Comment:
Please, try to avoid !! operator, maybe we can use requireActivity()?

Reply:
Yep, requireActivity() is a good chcoice.